news_items/043_units_element/index.html

```diff
@@ -0,0 +1,67 @@
+<style>
+  img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 15px;
+    margin-bottom: 15px;
+  }
+
+  pre {
+    width: 90%;
+    margin-left: auto;
+    margin-right: auto;
+    overflow: hidden;
+    margin-bottom: 1rem;
+    font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+    background: #f0f0f0;
+    padding: 1em;
+    padding-bottom: 0em;
+  }
+
+</style>
+
+<p>
+  Thanks to <a href="https://github.com/eliotwrobson">Eliot Robson</a> (UIUC), PrairieLearn now has a new <code
+    class="user-output">pl-units-input</code> element.
```

Comment:
Three UBC ugrads worked on a draft of this in #4801.  Does this share enough common history to acknowledge that?

Reply:
Happy to supply a string with names if so.

Reply:
Virtually everything has been rewritten, but I'll add a note about this.

Reply:
@cbm755 Added the two that are credited in commits here: https://github.com/PrairieLearn/PrairieLearn/pull/7302/commits/a88faa8ceced99a8770fb98f3d0ba5b7ba159461

Who is the third?

Reply:
Annie Geng, not sure I have a github link: the work was originally done on a private repo

Reply:
@cbm755 Added here: https://github.com/PrairieLearn/PrairieLearn/pull/7302/commits/bd519eca77086addbe686af0f3b9be3dfa2ed6fd

Reply:
No worries! Name reverted here https://github.com/PrairieLearn/PrairieLearn/pull/7302/commits/d85393bee1a8e71860b39819f86c583023520afc